test/e2e/pods.go

```diff
@@ -693,7 +693,8 @@ var _ = Describe("Pods", func() {
 									Port: intstr.FromInt(8080),
 								},
 							},
-							InitialDelaySeconds: 30,
+							InitialDelaySeconds: 15,
+							TimeoutSeconds:      30,
```

Comment:
I'm still unsure about bumping this from 1s to 30s. I think 10s is a reasonable start, and if it still fails, the node may jsut be overloaded.